pkg/logging/keys.go

```diff
@@ -0,0 +1,33 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+const (
+	// MARK: Common Keys
+	// > Note: we should only define common log keys here, that we want to ensure consistency between logging occurrences
+	// > No need to define every adhoc key.
+
+	ImageID = "imageID"
```

Comment:
Should these be suffixed with `Key` so `ImageIDKey` in this case?

suffixing with `Key` feels like more correct naming, but gets away from the const directly mapping its name to the underlaying string,